frontend/javascripts/admin/task/task_annotation_view.tsx

```diff
@@ -192,18 +197,19 @@ class TaskAnnotationView extends React.PureComponent<Props, State> {
     return (
       <div>
         <table>
-          <tbody>
+          <tbody className="task-annotation-view">
```

Comment:
this class is now assigned to tbody as well as td, which I find a bit unusual. also, why did you add it in the first place? I don't see it mentioned in the original issue nor in the pr description :thinking:

Reply:
right, I forgot to remove it from tbody, thank you!
I added the class because the alignment of the expanded rows differed from row to row and I thought it would be nice to make it consistent between all rows. If there was a bigger reason behind this or it was plain unnecessary I can remove it of course! Didnt mention it because I though its a minor change.

Reply:
> right, I forgot to remove it from tbody, thank you!

ah, ok, then it makes sense :+1: 

>  If there was a bigger reason behind this or it was plain unnecessary I can remove it of course! Didnt mention it explicitly because I thought its a minor change.

yes, that's fine! I only wanted to explain that I didn't get the change initially :)